src/Component/Result/Stats/Result.php

```diff
@@ -14,20 +14,15 @@
  */
 class Result
 {
+    use ResultTrait;
```

Comment:
What about re-adding getValue() for BC and declare it as deprecated?

```php
    /**
     * Get value by name.
     *
     * @deprcated
     *
     * @param mixed $name
     *
     * @return string|array|null
     */
    public function getValue($name)
    {
        return $this->getStatValue($name);
    }
```

Reply:
@thomascorthals ^

Reply:
That is so obvious that my facepalm must have registered on seismographs. 🤦‍♂️